easybuild/easyconfigs/x/XALT/XALT-2.8.1.eb

```diff
@@ -0,0 +1,60 @@
+name = 'XALT'
+version = '2.8.1'
```

Comment:
@samcmill Can we update this to `2.8.4` (and fix the SHA256 checksum), which includes important fixes (in particular, it should resolve the fork bomb issue I ran into...)

Reply:
Done